bioio/bio_image.py

```diff
@@ -125,6 +129,22 @@ class BioImage(biob.image_container.ImageContainer):
     by either instantiating the reader independently or using the `.reader` property.
     """
 
+    @staticmethod
+    def get_array_like_plugin() -> PluginEntry:
```

Comment:
I suggest moving this whole function to plugins.py

Reply:
Resolved in [0c6bb8e](https://github.com/bioio-devs/bioio/pull/47/commits/0c6bb8ecb19ceef7b1e17b50d9599ace6432b9dc)